common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1770,6 +1777,12 @@
     ],
     "sqlState" : "22P03"
   },
+  "INVALID_CHARACTER_IN_DATETIME_PATTERN" : {
```

Comment:
Could you convert it to sub-class of `INVALID_DATETIME_PATTERN`

Reply:
Done. Changed relevant tests also.